src/ExpressionEvaluator/Core/Source/ResultProvider/Helpers/DebuggerDisplayInfo.cs

```diff
@@ -1,19 +1,120 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
+using System;
 using Microsoft.VisualStudio.Debugger.Clr;
+using Microsoft.VisualStudio.Debugger.Evaluation;
 using Microsoft.VisualStudio.Debugger.Evaluation.ClrCompilation;
 
 namespace Microsoft.CodeAnalysis.ExpressionEvaluator
 {
-    internal struct DebuggerDisplayInfo
+    internal class DebuggerDisplayInfo
```

Comment:
>class [](start = 13, length = 5)

Should we change this to class?

Reply:
I suppose we could keep it a value type, but now that it contains logic and private data it seemed more appropriate to be a reference type. It make the initialization logic simpler in the constructors.